### PR TITLE
prepare for release

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: radiospectra/net/sources/callisto.py [hunk 1]
```diff
@@ -59,24 +62,24 @@ def pre_search_hook(cls, *args, **kwargs):
...
 
     @classmethod
     def _can_handle_query(cls, *query):
         """
-        Method the
-        `sunpy.net.fido_factory.UnifiedDownloaderFactory`
-        class uses to dispatch queries to this Client.
+        Method the `sunpy.net.fido_factory.UnifiedDownloaderFactory` class uses to dispatch queries.
+
+        to this Client.
```
Comment (ConorMacBride) [inline radiospectra/net/sources/callisto.py:82]:
```suggestion
```

File: radiospectra/_dev/__init__.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 """
-This package contains utilities that are only used when developing drms in a
-copy of the source repository.
+This package contains utilities that are only used when developing drms in a copy of the source.
+
+repository.
```
Comment (ConorMacBride) [inline radiospectra/_dev/__init__.py:4]:
```suggestion
```

File: radiospectra/net/sources/psp.py [hunk 3]
```diff
@@ -44,15 +45,17 @@ class RFSClient(GenericClient):
...
+    )
+    pattern = r"{}/{Wavelength}/{year:4d}/" r"psp_fld_l2_{Wavelength}_{year:4d}{month:2d}{day:2d}_v{:2d}.cdf"
 
     @classmethod
     def _check_wavelengths(cls, wavelength):
         """
-        Check for overlap between given wavelength and receiver frequency coverage defined in
+        Check for overlap between given wavelength and receiver frequency coverage defined in.
+
         `RECEIVER_FREQUENCIES`.
```
Comment (ConorMacBride) [inline radiospectra/net/sources/psp.py:59]: Here

File: radiospectra/net/sources/psp.py [hunk 4]
```diff
@@ -122,24 +122,25 @@ def search(self, *args, **kwargs):
 
     def post_search_hook(self, exdict, matchdict):
         """
-        This method converts 'rfs_hfr' and 'rfs_lfr' in the url's metadata
-        to the frequency ranges of for low and high frequency receivers.
+        This method converts 'rfs_hfr' and 'rfs_lfr' in the url's metadata to the frequency ranges.
+
+        of for low and high frequency receivers.
```
Comment (ConorMacBride) [inline radiospectra/net/sources/psp.py:127]:
```suggestion
```

File: radiospectra/net/sources/stereo.py [hunk 5]
```diff
@@ -3,24 +3,25 @@
...
 }
 
 
 class SWAVESClient(GenericClient):
     """
-    Provides access to STEREO `S-WAVES <https://swaves.gsfc.nasa.gov/swaves_instr.html>`__ radio
-    data `archive <https://solar-radio.gsfc.nasa.gov/data/stereo/>`__
+    Provides access to `STEREO S-WAVES <https://swaves.gsfc.nasa.gov/swaves_instr.html>`__ radio.
+
+    `data archive <https://solar-radio.gsfc.nasa.gov/data/stereo>`__.
```
Comment (ConorMacBride) [inline radiospectra/net/sources/stereo.py:24]:
```suggestion
```

**Review by ConorMacBride — COMMENTED** [review]
There seems to be lots of broken sentences at the end of doc strings?

**Comment by nabobalis** [comment]
> There seems to be lots of broken sentences at the end of doc strings?

I was experimenting with something, it did not work.

**Comment by nabobalis** [comment]
I will fix and push to main.

**Pull request merged**